### PR TITLE
Synapse 1.59

### PR DESCRIPTION
**Pull request by babolivier**
Preview: https://pr1335--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Push rules allow Matrix clients to define notification rules on a homeserver.
+One often reported issue with notification in Matrix is the fact that
+notifications are sent out when server ACLs, which define which server(s) can
+and cannot interact in a room, change. This is especially annoying during big
+waves of abuse, as there might be multiple servers that need to be banned from
+rooms, thus causing a lot of unneeded notifications.
+
+Synapse 1.59 now silences server ACL updates, by implementing the new push rule
+documented in
+[MSC3786](https://github.com/matrix-org/matrix-spec-proposals/pull/3786).
```
Comment (richvdh) [inline gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx:48]: only if the experimental option is set, I think?

File: gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Synapse 1.59 also allows third-party modules to validate and change the actions
+associated with an existing push rule via the [Module
+API](https://github.com/matrix-org/synapse/blob/d24a1486e5c5d69a8798a9d159fd9e06dfc8c3e3/synapse/module_api/__init__.py#L1360-L1420).
+This is helpful for modules wishing to, for example, configuring specific
+notification settings for new users register.
```
Comment (richvdh) [inline gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx:54]: this sentence seems to be missing some words, or something

File: gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+complete list of changes in this release. Also please have a look at the
+[upgrade
+notes](https://matrix-org.github.io/synapse/v1.59/upgrade#upgrading-to-v1590)
+for this version.
+
+Synapse is a Free and Open Source Software project, and we'd like to extend our
+thanks to everyone who contributed to this release, including (in no particular
+order) [Beeper](https://www.beeper.com/), [Dirk
+Klimpel](https://github.com/dklimpel), [Šimon
+Brandner](https://github.com/SimonBrandner),
```
Comment (richvdh) [inline gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx:80]: Šimon works for Element, so I'm not sure if it's appropriate to shout out to him here... otoh no real reason why not I guess!

Comment (babolivier) [reply gatsby/content/blog/2022/05/2022-05-18-synapse-1.59.mdx:80]: The biggest chunk of the work was done before they joined so I didn't think it was fair to consider it all an Element contribution.

**Review by richvdh — APPROVED** [review]
lgtm otherwise

**Pull request merged**